uniprot-search/src/main/resources/uniparc/search-fields.json

```diff
@@ -6,11 +6,11 @@
   },
   {
     "field": "accession",
-    "fieldValidRegex": "([O,P,Q][0-9][A-Z|0-9]{3}[0-9]|[A-N,R-Z]([0-9][A-Z][A-Z|0-9]{2}){1,2}[0-9])"
+    "fieldValidRegex": "([OPQ][0-9][A-Z0-9]{3}[0-9]|[A-NR-Z]([0-9][A-Z][A-Z0-9]{2}){1,2}[0-9])(-[0-9]+)?"
   },
   {
     "field": "isoform",
```

Comment:
accession and isoform have the same regular expression. Is it okay?

Reply:
good point. to me it makes sense only isoform has the `<dash number>` that terminates the id. I've updated the json and all tests pass, so i'll leave it.

if in future someone wants to change it, they can -- and they can add the appropriate tests.